src/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ if( OPENSSL_VERSION LESS "1.0.0" )
     #ignore openssl code warnings
     set_source_files_properties( ${openssl_SRCS} PROPERTIES COMPILE_FLAGS ${OPENSSL_IGNORE} )
     list( APPEND digidocpp_SRCS ${openssl_SRCS} )
+    list( APPEND digidocpp_DEFS NO_CMS )
```

Comment:
Either the compilation should fail if CMS support is not present or it would be nice if the additional feature is compiled with a "positive" approach. Meaning "WITH_CMS" and "#ifdef" rather than the negation. 

Also, I don't get it - the OpenSSL implementation is buggy, so the implementation does not work at all?